functions/go/export-terraform/terraformgenerator/resources.go

```diff
@@ -206,6 +218,8 @@ func (resource *terraformResource) attachReferences() error {
 		{kind: "ComputeRouter", path: []string{"spec", "routerRef", "name"}},
 		{kind: "ComputeAddress", customRetriever: singleComputeAddressRetriever([]string{"spec", "natIps"})},
 		{kind: "ComputeAddress", customRetriever: singleComputeAddressRetriever([]string{"spec", "reservedPeeringRanges"})},
+		//todo:awmalik@ - remove customerRetriver when this issue is addressed: https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/665
```

Comment:
nit: We should use `TODO` by convention.

Reply:
Ack